docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,240 @@
+        - The [client](https://appium.io/docs/en/2.0/intro/clients/) is responsible for sending commands to the server over the network, and receiving responses from the server as a result. You can choose the language of your choice to write the commands. For example, for Javascript [WebDriverIO](https://webdriver.io/) can be used as client.
+        > [Here](https://appium.io/docs/en/2.0/ecosystem/) you can read more about Appium ecosystem
+      - The advantage of this architecture is that it opens the possibility of running the server in a VM, and the client in a pipeline, enabling the tests to be ran automatically on scheduled basis as part of CI/CD pipelines. 
+
+### How to run mobile test locally on a Windows machine 
+In order to run locally such tests, you need two things:
+1. An emulator ([AVD - Android Virtual Devices](https://developer.android.com/studio/run/managing-avds)), which will represent the physical device.
+1. [Appium server](https://appium.io/docs/en/2.1/), which will redirect the commands from the test to your virtual device.
+
+#### a) Steps to create Android Virtual Device:
+1. Install Android Studio from [official link](https://developer.android.com/studio). 
+    > Note: At the time of writing the documentation, the latest version available was Android Studio Giraffe, 2022.3.1 Patch 2 for Window.
+    1. Set ANDROID_HOME environment variable to point to the installation path of Android SDK. i.e. ` C:Users\<user-name>\AppData\Local\Android\Sdk`
+1. Install Java Development Kit (JDK) from [official link](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). For the most recent devices JDK 9 is required, otherwise JDK 8 is required. Make sure you get the JDK and not the JRE.
+    1. Set JAVA_HOME environment variable to the installation path, i.e. ` C:\Program Files\Java\jdk-11`
+1. Create an AVD (Android Virtual Device):
+    1. Open Android Studio. From the Android Studio welcome screen, select **More Action -> Virtual Device Manager**, as instructed [here](https://developer.android.com/studio/run/managing-avds#:~:text=1%20Open%20the%20AVD%20Manager%20by%20clicking%20Tools,%20settings%2C%20such%20as%20the%20skin.%20See%20More.)
+    1. Click **Create Device**.
+    1. Choose a device definition with **Play Store** enabled. This is important, otherwise Teams cannot be installed on the device.
+    1. Choose a System image from the **Recommended** tab which includes access to Google Play services. You may need to install it before selecting it.
+    1. Start the emulator by clicking on the Run button from the Device Manage screen.
+    1. Manually install Microsoft Teams from Google Playstore on the device. 
```

Comment:
```suggestion
#### Creating an Android Virtual Device:

1. Install Android Studio from [official link](https://developer.android.com/studio). 
    > Note: At the time of writing the documentation, the latest version available was Android Studio Giraffe, 2022.3.1 Patch 2 for Window.
    1. Set ANDROID_HOME environment variable to point to the installation path of Android SDK. i.e. ` C:Users\<user-name>\AppData\Local\Android\Sdk`
1. Install Java Development Kit (JDK) from [official link](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). For the most recent devices JDK 9 is required, otherwise JDK 8 is required. Make sure you get the JDK and not the JRE.
    1. Set JAVA_HOME environment variable to the installation path, i.e. ` C:\Program Files\Java\jdk-11`
1. Create an AVD (Android Virtual Device):
    1. Open Android Studio. From the Android Studio welcome screen, select **More Action -> Virtual Device Manager**, as instructed [here](https://developer.android.com/studio/run/managing-avds#:~:text=1%20Open%20the%20AVD%20Manager%20by%20clicking%20Tools,%20settings%2C%20such%20as%20the%20skin.%20See%20More.)
    1. Click **Create Device**.
    1. Choose a device definition with **Play Store** enabled. This is important, otherwise Teams cannot be installed on the device.
    1. Choose a System image from the **Recommended** tab which includes access to Google Play services. You may need to install it before selecting it.
    1. Start the emulator by clicking on the Run button from the Device Manage screen.
    1. Manually install Microsoft Teams from Google Playstore on the device. 
```